src/utils.ts

```diff
@@ -52,3 +53,27 @@ export const getDocument = async (api: AdsApi, id: string): Promise<IOriginalDoc
     ? { notFound: true }
     : { doc: result.value.docs[0] };
 };
+
+export const getHasGraphics = async (api: AdsApi, bibcode: string): Promise<boolean> => {
+  const result = await api.graphics.query({
+    bibcode: bibcode,
+  });
+  return result.isErr() ? false : true;
+};
+
+export const getHasMetrics = async (api: AdsApi, bibcode: string): Promise<boolean> => {
```

Comment:
I'm thinking that these two helpers should maybe be moved into their respective api services instead.  Something like:

`api.metrics.hasMetrics(bibcode: string): Promise<boolean>`

Reply:
Done